ee_tests/src/specs/quickstart_analytic.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { browser, ExpectedConditions as until, $, $$ } from 'protractor';
+import * as support from './support';
+
+import { LandingPage } from './page_objects/landing.page';
+import { SpaceDashboardPage } from './page_objects/space_dashboard.page';
+import { SpacePipelinePage } from './page_objects/space_pipeline.page';
+
+/* Tests to verify the build pipeline */
+
+describe('Creating new quickstart in OSIO', () => {
+  let landingPage: LandingPage;
+
+  beforeEach( async () => {
+    await support.desktopTestSetup();
+    landingPage = new LandingPage(browser.params.target.url);
```

Comment:
we shouldn't use `target.url` since --baseUrl is already set and all paths are relative to base url